packages/headless/src/database/mongodb/user/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import { UserEntity } from 'src/entity/user';
+import { IUserDB } from 'src/modules/user/repository/user.db.interface';
+import { UserModel } from './user.model';
+
+@Injectable()
+export class UserAuthDB implements IUserDB {
+  async save(body: UserEntity): Promise<UserEntity | null> {
+    const isExist = await UserModel.findOne({ phone: body.phone }).lean();
+    if (isExist) {
+      return null;
+    }
+    const user = (await UserModel.create(body)).toJSON();
+    delete user.password;
```

Comment:
you can remove this from here and do it in repository to reduce redundant code in multiple database.